### PR TITLE
fix custom sort

### PR DESCRIPTION
**Pull request by emptyrivers**

### CODE REVIEW

File: WeakAuras/RegionTypes/DynamicGroup.lua [hunk 1]
```diff
@@ -329,7 +329,7 @@ local sorters = {
     if #events > 0 then
       sortOn = {}
       for _, event in ipairs(events) do
-        events[event] = true
```
Comment (emptyrivers) [inline WeakAuras/RegionTypes/DynamicGroup.lua:332]: :( i can't believe i didn't notice this

**Comment by Stanzilla** [comment]
did you test this in-game?

File: WeakAuras/Modernize.lua [hunk 2]
```diff
@@ -1633,7 +1633,7 @@ function Private.Modernize(data)
 
   if data.internalVersion < 62 then
     if data.regionType == "dynamicgroup" then
-      if data.sort == "CUSTOM" and type(data.sortOn) ~= "string" then
+      if data.sort == "custom" and type(data.sortOn) ~= "string" then
```
Comment (Stanzilla) [inline WeakAuras/Modernize.lua:1636]: do you need another modernize to fix this up?

Comment (InfusOnWoW) [reply WeakAuras/Modernize.lua:1636]: Yeah you need to do that in a new modernize version, so that the code that we wrongly didn't convert gets converted.

**Review by Stanzilla — APPROVED** [review]

**Pull request merged**